shell/android-studio/reicast/src/main/java/com/reicast/emulator/emu/GL2JNIView.java

```diff
@@ -64,307 +65,308 @@
-
-	float editLastX = 0, editLastY = 0;
-
-	@Override public boolean onTouchEvent(final MotionEvent event) 
-	{
-		JNIdc.show_osd();
-
-		scaleGestureDetector.onTouchEvent(event);
-
-		float ty  = 0.0f;
-		float scl  = getHeight()/480.0f;
-		float tx  = (getWidth()-640.0f*scl)/2;
-
-		int   rv  = 0xFFFF;
-
-		int   aid = event.getActionMasked();
-		int   pid = event.getActionIndex();
-
-		if (editVjoyMode && selectedVjoyElement != -1 && aid == MotionEvent.ACTION_MOVE && !scaleGestureDetector.isInProgress()) {
-			float x = (event.getX()-tx)/scl;
-			float y = (event.getY()-ty)/scl;
-
-			if (editLastX != 0 && editLastY != 0) {
-				float deltaX = x - editLastX;
-				float deltaY = y - editLastY;
-
-				vjoy_d_custom[selectedVjoyElement][0] += isTablet() ? deltaX * 2 : deltaX;
-				vjoy_d_custom[selectedVjoyElement][1] += isTablet() ? deltaY * 2 : deltaY;
-
-				requestLayout();
-			}
-
-			editLastX = x;
-			editLastY = y;
-
-			return true;
-		}
-
-		for(int i=0;i<event.getPointerCount();i++)
-		{ 
-			float x = (event.getX(i)-tx)/scl;
-			float y = (event.getY(i)-ty)/scl;
-			if (anal_id!=event.getPointerId(i))
-			{
-				if (aid==MotionEvent.ACTION_POINTER_UP && pid==i)
-					continue;
-				for(int j=0;j<vjoy.length;j++)
-				{
-					if(x>vjoy[j][0] && x<=(vjoy[j][0]+vjoy[j][2]))
-					{
+        ethd = new EmuThread(!Emulator.nosound);
+
+        touchVibrationEnabled = prefs.getBoolean(Config.pref_touchvibe, true);
+        vibrationDuration = prefs.getInt(Config.pref_vibrationDuration, 20);
+
+        this.setLayerType(prefs.getInt(Config.pref_rendertype, LAYER_TYPE_HARDWARE), null);
+
+        vjoy_d_custom = VJoy.readCustomVjoyValues(context);
+
+        scaleGestureDetector = new ScaleGestureDetector(context, new OscOnScaleGestureListener());
+
+        // This is the game we are going to run
+        fileName = newFileName;
+
+        if (Emulator.nativeact) {
+            if (GL2JNINative.syms != null)
+                JNIdc.data(1, GL2JNINative.syms);
+        } else {
+            if (GL2JNIActivity.syms != null)
+                JNIdc.data(1, GL2JNIActivity.syms);
+        }
+
+        JNIdc.init(fileName);
+
+        // By default, GLSurfaceView() creates a RGB_565 opaque surface.
+        // If we want a translucent one, we should change the surface's
+        // format here, using PixelFormat.TRANSLUCENT for GL Surfaces
+        // is interpreted as any 32-bit surface with alpha by SurfaceFlinger.
+        if(translucent) this.getHolder().setFormat(PixelFormat.TRANSLUCENT);
+
+        if (prefs.getBoolean(Config.pref_egl14, false)) {
+            setEGLContextFactory(new GLCFactory14.ContextFactory());
+            setEGLConfigChooser(
+                    translucent?
+                            new GLCFactory14.ConfigChooser(8, 8, 8, 8, depth, stencil)
+                            : new GLCFactory14.ConfigChooser(5, 6, 5, 0, depth, stencil)
+            );
+            GLES20.glEnable(GLES20.GL_DEPTH_TEST);
```

Comment:
As far as I can see, this is the only real functionality change, the rest is only whitespace changes.
I can't comment on this, as I don't know enough about GLES or Android.